src/test/javascript/integration/historical_epoch_direction_queries_spec.js

```diff
@@ -45,15 +41,15 @@ describe(testName, function () {
     it('posts historical items to ' + channel, function (done) {
         utils.postItemQ(getFormattedUrl(earliestTime))
             .then(function (value) {
-                items.push(value.response.headers.location)
+                items.push(fromObjectPath(['response', 'headers', 'location'], value))
                 return utils.postItemQ(getFormattedUrl(earliestTime.add(1, 'years')));
             })
-            .then(function (value) {
-                items.push(value.response.headers.location)
+            .then(function (value1) {
```

Comment:
Same question as above. Why not use `value` for the arguments in the promise chain?